pkg/controller/cassandracluster/generator.go

```diff
@@ -784,3 +790,25 @@ func createCassandraContainer(cc *api.CassandraCluster, status *api.CassandraClu
 
 	return cassandraContainer
 }
+
+func createBackRestSidecarContainer(cc *api.CassandraCluster, status *api.CassandraClusterStatus,
```

Comment:
why do you have this code as a change ? It should be already present in your branch if you created it on top of the master branch